easybuild/easyconfigs/t/TRIQS/TRIQS-3.1.0-gofb-2021a.eb

```diff
@@ -25,22 +25,83 @@ dependencies = [
     ('Clang', '13.0.1'),
     ('FFTW', '3.3.9'),
     ('HDF5', '1.12.1'),
+    ('mpi4py', '3.1.3'),
+    ('SciPy-Stack', '2022a'),
 ]
 
 builddependencies = [
     ('CMake', '3.22.1'),
-    ('mpi4py', '3.1.3'),
-    ('SciPy-Stack', '2021a'),
+    ('pytest', '7.0.1'),
 ]
 
-multi_deps = {'Python': ['3.8', '3.9', '3.10']}
+multi_deps = {'Python': ['3.9']}
```

Comment:
One of the extension is not compatible with Python 3.10 because it uses a depreciated module removed in Python 3.10. Also, I got installation error with multi_deps when using multiple Python installations. So I choose the 3.9

Reply:
It should then also support 3.8, which is the default python at the moment.